### ISSUE TITLE
Unable to resolve modified source dependency in sbt

### ISSUE THREAD

**Issue by sagariEduri22** [Activiti/Activiti#941]
I have downloaded the activiti source code and trying to modify something. After that, I want to build and publish the modified jar file to my local artifactory so that I can use this new jars in my scala application. I was able to publish to artifactory using `mvn deploy -Pdistro -DskipTests`. However, I after adding the dependency in my sbt file, I am getting the following error. 

> [error] unresolved dependency: org.activiti#activiti-engine;5.22.0-SNAPSHOT: 	local-          > snapshots: no ivy file nor artifact found for org.activiti#activiti-root;5.22.0-SNAPSHOT

Here is my dependency:

` "org.activiti" % "activiti-engine" % "5.22.0-SNAPSHOT"`

I have already asked this question in stackoverflow and activiti forums, but haven't got any solution. I am stuck because of this. Can anyone help me to get the dependency resolved? 

I tried adding 
`"org.activiti" % "activiti-root" % "5.22.0-SNAPSHOT"  pomOnly()` 
but still the same issue

**Comment by tijsrademakers** [comment]
when running this Maven command from the Activiti root, all Activiti artefacts should be available in your Maven repository. Is there a reason why you are not using the released 5.21.0 version?

**Comment by sagariEduri22** [comment]
yes,I have all the artefacts in maven repository.I just downloaded the latest source code.No specific reason to go for this.Is there any issues w.r.t the 5.22.0-SNAPSHOT ?

would please recommend the version,so that i can proceed with that.

**Comment by tijsrademakers** [comment]
you can use any version you want, but many of the Activiti users would choose the stable release version, so 5.21.0. There are no issues with 5.22.0-SNAPSHOT, but it's not a stable version.

**Comment by sagariEduri22** [comment]
Okay,Thank you Tijs.I will try with 5.21.0 and ll get back if i face any issues there.
